avm/res/network/nat-gateway/version.json

```diff
@@ -1,7 +1,7 @@
 {
-    "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
-    "version": "1.2",
-    "pathFilters": [
-        "./main.json"
-    ]
+  "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
+  "version": "1.3",
```

Comment:
damn

Reply:
Jap. We eventually have to work on the `settings.json` and `extension.json`. Can't say that I changed anything, so not sure why it is changing  or even what would be the intended one. The good news: It doesn't matter. The other good news: Comparing it with this [PR](https://github.com/Azure/bicep-registry-modules/pull/4969/files), we share the same setting 😄

Reply:
It was more about the 1.x version. We know about it but I wanted to seize the opportunity to curse anyway. Feel free to resolve

Reply:
ah pardon. Right. Curses!